crmsh/cibconfig.py

```diff
@@ -3137,7 +3137,7 @@ def modified_elems(self):
 
     def get_elems_on_ra_type(self, spec):
```

Comment:
Maybe rename the function name as well, something like,
```suggestion
    def get_elems_of_ra_partial_search(self, spec):
```